source/blender/gpu/intern/gpu_codegen.c

```diff
@@ -55,6 +55,11 @@
 #include <string.h>
 #include <stdarg.h>
 
+#ifndef ARRAYSIZE
```

Comment:
Is it possible to include `BLI_utildefines.h` for this macro ?

Reply:
Yes, done